### ISSUE TITLE
mocha-webpack doesn't support webpack environment arguments env.*

### ISSUE THREAD

**Issue by paztis** [zinserjan/mocha-webpack#156]
When trying to pass webpack environment arguments like this 'mocha-webpack --webpack-config ./build/webpack.config.build.js  --env.arg1=XXX', an error is thrown: 'Unknown argument: env'

**Comment by paztis** [comment]
Great thanks. With --webpack-env can we pass objects? In weback we can pass --env.arg1=value1 --env.arg2=value2 and we get back the object {arg1: "value1", arg2:" value2"} in webpack config function. Can we use here --webpack-env.arg1 ?

**Comment by zinserjan** [comment]
Yep, see https://github.com/zinserjan/mocha-webpack/pull/162/files#diff-338fd8b79b74bbe4d2089293ce2eb187

Implementation is copy & paste from webpack ;)

**Comment by paztis** [comment]
Thanks a lot.
